tooling/nargo_fmt/tests/expected/expr.nr

```diff
@@ -0,0 +1,35 @@
+// Test some empty blocks.
+fn qux() {
+    {}
+    {
+    /* a block with a comment */
+    }
+    {}
+    {
+    // A block with a comment.
+    }
+}
+fn foo_return() {
+    "yay"
+}
+fn fooblock() {
+    {
+        "inner-block"
+    }
+}
+fn fooblock() {
+    {
+        {
+            {
+                "inner-block"
+            }
+        }
+    }
+}
+fn comment() {
+    // this is a test comment
+        1
+}
+fn only_comment() {
+// Keep this here
```

Comment:
It looks like we expect an extra indent on the 1 and no extra newline between these two functions. Are the "input" and "expected" files switched for this test?